src/components/Segment/Segment.tsx

```diff
@@ -43,7 +43,7 @@ class Segment extends UIComponent<ReactProps<SegmentProps>, any> {
 
     return (
       <ElementType {...unhandledProps} className={classes.root}>
-        {childrenExist(children) ? children : Slot.create(content)}
+        {childrenExist(children) ? children : Component.create(content)}
```

Comment:
another misleading point, as result of `Component.create()` is actually an element, not a component

Reply:
Per above comments, what if this were `Box.create()`?

Reply:
agreed, see https://github.com/stardust-ui/react/pull/713/#discussion_r247552617

Reply:
renamed to `Box`